### DISCUSSION TITLE
how to get gui Authentication required window on leftwm

### DISCUSSION THREAD

**Discussion by NicTanghe** [leftwm/leftwm#1023] [Q&A]
Hello i have a progam that installs things and when i run it as admin it installs the programs incorrectly.

So the method i should use is run the program as a user and give my sudo password in a window that pops up.

I can`t get the window to popup in LEftwm though so I`m wondering what i need to run at startup for this to stil happen.

(to get the same behavior i get on GNOME)

**Comment by NicTanghe** [comment]
Dunst doesn't seem to do this.

**Reply by NicTanghe** [reply]
(wel after fiddeling about it seems that i just cant get the gnome version working so i installed another one.)
(this was ages ago_

**Comment by VuiMuich** [comment]
I believe this might be a setting for PAM, but I can't find any guide how to explicitly set this up at the moment.
But I believe on my Manjaro install the `pinentry` (https://gnupg.org/related_software/pinentry/) tool is used for those dialogs.

**Comment by NicTanghe** [comment]
Just install an polkit manager ?
